Docs 3379 update encryption articles (#1617)
truenas/documentation: content/_includes/ReplicatedEncryptedUnlock.md

4. Click **Save**.

### Method 3: Replicate Key Encrypted Dataset/zvol.

1. Go to **Datasets** on the system you are replicating from. 

>    Select the dataset encrypted with a key, click **Export Key** on the **ZFS Encryption** widget to export the key for the dataset.

Reviewer: ```suggestion
   Select the dataset encrypted with a key, then click **Export Key** on the **ZFS Encryption** widget to export the key for the dataset.
```

2. Apply the JSON key file or key code to the dataset on the system you replicated the dataset to. 

>    Either download the key file, open that file in a text editor and change the *pool name/dataset* part of the string to the *pool name/dataset* for the receiving system. For example, replicating from *tank1/dataset1* on the replicate-from system to *tank2/dataset2* on the replicate-to system. Or you can copy the key code provided in the **Key for *dataset*** window.

Reviewer: ```suggestion
* Option 1: Download the key file and open it in a text editor. Change the *pool name/dataset* part of the string to the *pool name/dataset* for the receiving system. For example, replicating from *tank1/dataset1* on the replicate-from system to *tank2/dataset2* on the replicate-to system. 
* Option 2: Copy the key code provided in the **Key for *dataset*** window.
```

3. On the system receiving the replicated pool/dataset, select the receiving dataset and click **Unlock**. 

4. Unlock the dataset. 

>    Either clear the **Unlock with Key file** checkbox, paste the key code into **Dataset Key** field (if there is a space character at the end of the key, delete the space), or select the downloaded Key file that you edited.

Reviewer: ```suggestion
   Either clear the **Unlock with Key file** checkbox, paste the key code into the **Dataset Key** field (if there is a space character at the end of the key, delete the space), or select the downloaded Key file that you edited.
```
